data/json/professions.json

```diff
@@ -1030,6 +1030,110 @@
       "female": { "entries": [ { "item": "bra" }, { "item": "panties" } ] }
     }
   },
+  {
+    "type": "profession",
+    "id": "physicist",
+    "name": "Physicist",
+    "description": "Your degree in physics might not have been theoretical, but the Cataclysm purely was.  Now that your antithesis has been irrefutably debunked, you might want to switch over to a career with higher chances of survival.",
+    "points": 3,
+    "skills": [ { "level": 4, "name": "mechanics" }, { "level": 4, "name": "electronics" } ],
+    "proficiencies": [ "prof_electromagnetics", "prof_pneumatics" ],
+    "items": {
+      "both": {
+        "entries": [
+          { "item": "dress_shirt" },
+          { "item": "tie_necktie" },
+          { "item": "tieclip" },
+          { "item": "pants", "variant": "pants_stone" },
+          { "item": "socks" },
+          { "item": "dress_shoes" },
+          { "item": "coat_lab" },
```

Comment:
I don't think Physicist should have lab coat, it is to pervert spilling after all.